apps/api/src/app/portfolio/calculator/portfolio-calculator.ts

```diff
@@ -130,17 +138,27 @@ export abstract class PortfolioCalculator {
       });
 
     this.redisCacheService = redisCacheService;
-    this.useCache = useCache;
+    this.useCache = false; // useCache;
     this.userId = userId;
 
-    const { endDate, startDate } = getInterval(dateRange);
```

Comment:
`dateRange` is unused after this change leading to max performance regardless of the selected date range 🤔